CHANGELOG.md

```diff
@@ -33,8 +33,11 @@
 - Bump firebase config to v21.0.1
 - Add facility protocol and group UUID in demo facility
 - [In Progress: 11 Jan 2022] Implement adding NHID in `PatientEditScreen`
-- [In Progress: 17 Jan 2022] Implement adding scan support for BpPassport in `EditPatientScreen`
 
+### Changes
+- Implement adding scan support for BpPassport in `EditPatientScreen`
+  - Support scanning list of unique Bp passports in `EditPatientScreen`
```

Comment:
There is an extra space here, can we refactor this please?

Reply:
The space is intended because it comes under the change

Reply:
We don't need to list it here, since this feature is not out yet for users yet. We can move the second line to internal change as it's a bug fix.